src/main/resources/sam.conf

```diff
@@ -243,3 +243,52 @@ janitor {
   trackResourceTopicId = ${?JANITOR_TRACK_RESOURCE_TOPIC_ID}
 }
 
+terra.support.emails = [
```

Comment:
this new config should replace [this script](https://github.com/broadinstitute/dsp-scripts/blob/master/terra-support-sam-policies/terra-support-sam-policies.py)

Reply:
see also https://github.com/broadinstitute/terra-helmfile/pull/5706